website/static/js/registrationEditorExtensions.js

```diff
@@ -142,7 +142,19 @@ var Uploader = function(data) {
     $.extend(self, data);
 };
 
+var AuthorImport = function(data, $root) {
+    var self = this;
+
+    self._orig = data;
+    self.contributors = $root.contributors();
+    self.currentQuestion = $root.currentQuestion;
+    self.currentQuestion($root.find('authorship'));
```

Comment:
The Import Contributors button could be pressed from anywhere on the page but the `authorsDialog` uses `currentQuestion.setValue` to fill out the field.